### DISCUSSION TITLE
Any new issues?

### DISCUSSION THREAD

**Discussion by ghost** [Crypto-TII/claasp#67] [General]
Hello, My name is Ahmed, I am a junior crypto engineer, and i am wandering if there is any new open issue to work on.

**Comment by juaninf** [comment]
Hi @AhmeedMalek, sorry for the late reply. A good starting point is to understand the library's structure. To understand the library's structure, you might start by adding a cipher to it. For example, you could add the well-known Blowfish cipher, which is used by bcrypt.
